docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic.TensorFlow
+{
+    class TextClassification
+    {
+        public const int MaxSentenceLenth = 600;
+        /// <summary>
+        /// Example use of the TensorFlow sentiment classification model.
+        /// Download the model from 
+        /// https://github.com/dotnet/machinelearning-testdata/blob/master/Microsoft.ML.TensorFlow.TestModels/sentiment_model
+        /// The model is in 'SavedModel' format. For further explanation on how was the `sentiment_model` created 
+        /// c.f. https://github.com/dotnet/machinelearning-testdata/blob/master/Microsoft.ML.TensorFlow.TestModels/sentiment_model/README.md
+        /// </summary>
+        public static void ScoringWithTextClassificationModelSample()
```

Comment:
>ScoringWithTextClassificationModelSample [](start = 27, length = 40)

this project is for API reference samples. You'll need to reference this sample from the extension that it documents. See Ivan's comment on the other file.